packages/emf/backend/sirius-components-emf/src/main/java/org/eclipse/sirius/components/emf/handlers/EditingContextSearchContentEventHandler.java

```diff
@@ -49,15 +49,15 @@
  * @author gcoutable
  */
 @Service
-public class EditingContextOmniboxCommandsEventHandler implements IEditingContextEventHandler {
+public class EditingContextSearchContentEventHandler implements IEditingContextEventHandler {
```

Comment:
Renamed to reflect the actual purpose of the handler: it returns commands for elements contained in the editing context. Other handlers could return different commands.